src/Engine/ProtoCore/Lang/BuiltInFunctionEndPoint.cs

```diff
@@ -1178,36 +1181,36 @@ internal static StackValue[] GenerateRangeByAmount(decimal start, long amount, d
                        Resources.kInvalidAmountInRangeExpression);
                     return null;
                 }
-                decimal stepsize = new decimal(svStep.ToDouble().RawDoubleValue);
+                decimal stepsize = new decimal(svStep.ToDouble().DoubleValue);
                 return GenerateRangeByAmount(start, amount, stepsize, isIntRange);
             }
             else
             {
-                switch (svOp.opdata)
+                switch (svOp.IntegerValue)
```

Comment:
Right. I'm still thinking if we should add more AddressType to make type-safety, and my current answer is yes. For example, you may notice `IntegerValue` will be used in representing the depth of stack, the return address, the stack frame pointer and so on. The final goal is to make `AddressType` be private.